src/lostcity/engine/GameMap.ts

```diff
@@ -243,10 +276,11 @@ export default class GameMap {
                 for (let z: number = 0; z < GameMap.Z; z++) {
                     const absoluteZ: number = z + mapsquareZ;
 
+                    if (!this.members && !this.isFreeToPlay(absoluteX, absoluteZ) && !this.bordersFreeToPlay(absoluteX, absoluteZ)) {
```

Comment:
likewise this whole thing can be flipped if it makes the logic clearer

```suggestion
                    if (this.members || this.isFreeToPlay(absoluteX, absoluteZ) || this.bordersFreeToPlay(absoluteX, absoluteZ)) {
```